src/lib/components/Navigation/Navbar/Navbar.svelte

```diff
@@ -1,13 +1,15 @@
 <script lang="ts">
+	import { messages } from '$lib/i18n';
+
 	let { title, username }: { title: string; username: string } = $props();
 
-	let greeting = $derived(`Welcome ${username}!`);
+	let message = $derived(messages.greeting({ name: username }));
 </script>
 
 <header class="navbar justify-between bg-base-200 px-4">
 	<h2 class="prose prose-xl">Hestia</h2>
 	<h1 class="prose prose-2xl">{title}</h1>
```

Comment:
I know the title is a prop but did you use a translation when you call this component on the title prop?

Reply:
No since the title would be tenant name